### PR TITLE
Fix some minor incorrect anchor links

### PR DESCRIPTION
**Pull request by imsheridan**
This PR is to fix some incorrect anchor links as below.
- Fix the incorrect anchor name "PrepareLinux", which cause cannot direct to correct section with link: https://www.tensorflow.org/install/install_sources#PrepareLinux 
- Fix https://www.tensorflow.org/install/install_linux#CommonInstallationProblems with https://www.tensorflow.org/install/install_linux#common_installation_problems since no this anchor defined actually;

### CODE REVIEW

File: tensorflow/docs_src/install/install_sources.md [hunk 1]
```diff
@@ -76,9 +76,9 @@ issue the following command:
...
 
 Next, you must prepare your environment for
-[Linux](#PrepareLinux)
+[Linux](#prepare_environment_for_linux)
 or
-[macOS](#PrepareMac)
+[macOS](#prepare_environment_for_macos)
 
 
 <a name="#PrepareLinux"></a>
```
Comment (jhseu) [inline tensorflow/docs_src/install/install_sources.md]: Not sure I understand. Isn't the link name set here?

Comment (imsheridan) [reply tensorflow/docs_src/install/install_sources.md]: Oh I didn't notice the link name here. But actually it not works as you can see above links event it has set this link name. 

I mean https://www.tensorflow.org/install/install_sources#PrepareLinux cannot direct to the correct anchor section but https://www.tensorflow.org/install/install_sources#prepare_environment_for_linux can.

I can remove this link name "PrepareLinux" line or keep this but fix in "PrepareLinux" way. Is there any preference for you? If either would be ok, I'll remove this "PrepareLinux" line.

Comment (imsheridan) [reply tensorflow/docs_src/install/install_sources.md]: @jhseu Ah I found why the pre-defined anchor not works, it has a wrong # in the anchor name. Have pushed a fix to fix this incorrect anchor name.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jhseu: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by jhseu — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jhseu: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Pull request merged**